sdk/maps/azure-maps-render/swagger/CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Release History
```

Comment:
Why do we need this file? The CHANGELOG.md should be updated in https://github.com/khmic5/azure-sdk-for-java-1/blob/feature/katherine/renderenumdeprecation/sdk/maps/azure-maps-route/CHANGELOG.md